test/framing/FrameTransportTest.cpp

```diff
@@ -48,28 +48,6 @@ TEST(FrameTransport, CloseWithError) {
   transport->closeWithError(std::runtime_error("Uh oh"));
 }
 
-TEST(FrameTransport, SimpleEnqueue) {
```

Comment:
This test might be still useful that if no FrameProcessor is set, we can check that the frames are successfully dropped and no crash etc happens.